cuda/ubi8-python-3.8/Dockerfile

```
@@ -81,6 +81,48 @@ RUN yum install -y \
     && yum clean all \
     && rm -rf /var/cache/yum/*
 
+# Install CUDA devel from:
+# https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/11.4.2/ubi8/devel/Dockerfile
+ENV NV_NVPROF_VERSION 11.4.120-1
```

Missing ENV from https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/11.4.2/ubi8/devel/Dockerfile#L6

```suggestion
ENV NV_CUDA_LIB_VERSION 11.4.2-1
ENV NV_NVPROF_VERSION 11.4.120-1
```